cmd/status.go

```diff
@@ -42,7 +46,8 @@ var statusCmd = &cobra.Command{
 		}
 		status, err := c.Status(context.Background())
 		if err != nil {
-			return err
+			logrus.WithError(err).Debug("failed to get status")
```

Comment:
I am still of the opinion that this should be an error as it practically is the reason we weren't able to the the one thing this command does. It will also help someone with debugging what actually happened. 
Looking at this more and more ... we are basically saying that any error will be because a test will not be running which ... is probably true, but maybe there are other possible failures ... and saying "no test running" might be a little bit presumptuous ...

Reply:
@MStoykov Yes, that's why I left the log line in debug mode. One can still running `k6 -v status` to see the actually error message.

Reply:
yeah but the user will need to do it and know that they need to do it ... for the benefit of not showing one more line  ... 

IMHO this really doesn't matter much as I doubt all that many people either one of those commands, but if anyone ever runs into a situation where their tests are running (for example in a docker) and they run this and it tells them that "test is not running", nobody will think about adding `-v` before writing an issue or on the community forum.

Reply:
@MStoykov maybe make the error message more details, something like `no test are running, or some internal error occurs, please run with "-v" for more details.